pallet/src/main/java/com/strategyobject/substrateclient/pallet/annotations/Pallet.java

```diff
@@ -0,0 +1,22 @@
+package com.strategyobject.substrateclient.pallet.annotations;
+
+import lombok.NonNull;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates the pallet which represents a proxy to the blockchain's pallet.
+ * For interfaces annotated with it processor will generate proper implementations.
```

Comment:
The processor will generate proper implementations for the interfaces annotated with this method.